modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -576,6 +576,30 @@ Starting from 2025.04, using the error messages programmatically is deprecated.
 Because error messages might change in a future release, please parse or use GQLSTATUS error codes in your application code.
 For details, see link:{neo4j-docs-base-uri}/status-codes/current/errors/gql-errors/[Status Codes for Errors & Notifications -> List of GQLSTATUS error codes].
 
+== Cypher 25
```

Comment:
I don't have strong opinion about this section. On one hand, it can be helpful as we cover procedures in this manual and list other removed procedures on this page. On the other hand, the first group of removed procedures was taken away in the specific Neo4j release (2025.01); and today we list procedures removed from the new Cypher version (which is available starting from 2025.06).  In this case, all removals and deprecations related to Cypher should be covered in the Cypher manual. 
So, if this section causes questions and troubles instead of helping, I am happy to remove it

Reply:
To add labels _Removed in Cypher 25_ to the table _Deprecated procedures_ https://neo4j.com/docs/operations-manual/current/changes-deprecations-removals/#_procedures on this page?